packages/nimbus/vite.config.ts

```diff
@@ -48,7 +48,7 @@ const external = [
   "react/jsx-runtime",
 
   // UI frameworks & styling.
-  "@chakra-ui/react",
+  new RegExp("@chakra-ui/react?[^.].*$"),
```

Comment:
This is the main change that made the `DataTable` work in the MC, otherwise we were bundling all the chakra subdirectories (e.g. `@chakra-ui/react/styled-system`) with `nimbus`, but needed to add `@chakra-ui/react` as a dep in the consuming app. This led to the bundled chakra methods not working with the installed chakra library.

This enables `@chakra-ui/react` to be a peer dependency as intended.

The other option would be to totally bundle chakra with nimbus like we do with `react-aria`.  I'm starting to think we should either make both libraries peer-deps or bundle both, but that's a discussion for another time.